src/v1/events/types.ts

```diff
@@ -30,6 +30,7 @@ export type InsertEventParams = {
     amount: number;
   };
   searchQuery?: string;
+  referrerPageId?: string;
```

Comment:
This type definition misses `page_id`. Could you add it too? 
<img width="642" alt="image" src="https://user-images.githubusercontent.com/85863763/165871286-cbdd3750-c242-44e5-8eae-f459c2f4ebb4.png">
from https://moloco-rmp.readme.io/reference/rmpeventapi_postuserevent